src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -27,6 +27,7 @@
 
 import polars as pl
 
+import pymovements as pm  # pylint: disable=cyclic-import
```

Comment:
This PR includes an EventsDataFrame in a GazeDataFrame, which leads to circular dependencies when importing both.

Doing something like
```python
from pymovements.gaze.gaze_dataframe import GazeDataFrame
```
results in a hard python circular import error.

I have found a workaround by importing the pymovements top-level package, but then pylint is not happy with me again.

Right now I just can't find a way to untie this knot. As this is just a linting issue, I would like to create a follow-up issue on how to solve this problem and leave it like this for this PR.